corgi/monitor/consumer.py

```diff
@@ -206,16 +205,19 @@ def sbomer_complete(event: Event) -> bool:
     @staticmethod
     def pyxis_manifest_create(event: Event) -> bool:
         logger.info(f"Handling UMB message for pyxis manifest {event.message.id}")
-        json.loads(event.message.body)
+        message = json.loads(event.message.body)
+
         try:
-            pass
-            # Temporarily disabled due to slowness and lots of duplicate data
-            # slow_fetch_pyxis_manifest.delay(
-            #     message["entityData"]["_id"]["$oid"],
-            # )
+            created_by = message["entityData"]["created_by"]
```

Comment:
Should we also skip processing if the entityData/created_by field is missing, or you design it to fail in that case? I guess either is OK.

Perhaps we should add a test for that case also?